qiskit/visualization/latex.py

```diff
@@ -550,7 +550,7 @@ def _build_measure(self, node, col):
 
         self._latex[wire1][col] = "\\meter"
         if self.cregbundle:
-            self._latex[wire2][col] = "\\dstick{_{_{%s}}} \\cw \\cwx[-%s]" % (
+            self._latex[wire2][col] = "\\dstick{{_{{_{{{}}}}}}} \\cw \\cwx[-{}]".format(
                 str(cregindex),
                 str(wire2 - wire1),
```

Comment:
Probably best to revert this one (and the next) to %-formatting - the f-string form is illegible.